meta-ostro/conf/local.conf.sample

```diff
@@ -291,6 +291,18 @@ SUPPORTED_RECIPES_CHECK = "fatal"
 #
 #require conf/distro/include/ostro-os-production.inc
 
+# systemd-bootchart is a useful tool to analyze and optimize a system
+# boot time. The tool is available in Ostro and needs to be activated
+# by a kernel command-line parameter which requires to build a new
+# image. You also need to make sure that you enable CONFIG_SCHEDSTATS=y
+# (that will also set CONFIG_SCHED_INFO=y) in your kernel, this option
+# is not enabled by default. For instructions on how to modify an Ostro
+# kernel, please check doc/howtos/modifying-ostro-kernel.rst.
```

Comment:
meta-ostro-bsp/recipes-kernel/linux/ might be the right place, similar to meta-intel-iot-security/meta-security-framework/recipes-kernel/linux. Just remember that it cannot assume that the kernel is called "linux-yocto". We also have "linux-yocto-edison", so a "linux-yocto_*.bbappend" does not work for all kernel, and some downstream user might use something which is not a Yocto kernel at all. "linux-*.bbappend" as in meta-intel-iot-security works, but is a bit hacky (see commit log messages there).

The alternative to the linux-*.bbappend would be to extend the file search path in the layer conf for all recipes, but that might have unexpected consequences, so I'd not go down that route.